tests/lib/state_test.py

```diff
@@ -315,6 +315,89 @@ def test_state_iface_routes_order():
             reverse_route_state.config_iface_routes)
 
 
+def test_state_merge_config_add():
```

Comment:
As `absent` logic is included in the production code, there is a need to add a test reflecting that any absent record is removed on merge.

Reply:
Thanks. Added.